### DISCUSSION TITLE
On "Your Stars" page, clicking "Next" at the bottom automatically keeps the next page also at the bottom, forcing you to scroll up

### DISCUSSION THREAD

**Discussion by Saloframes** [community/community#69857] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When on Your Stars page, clicking "next" at the bottom does not bring the next page back to the top. It just keeps it at the bottom, forcing you to scroll back up. This only started happening recently. Happens both in Firefox and Chrome on Windows, even with all extensions disabled. Video of behavior below:

![Animation](https://github.com/community/community/assets/5899823/adc20b69-72cc-4b88-9bf7-73bec43bff07)

**Comment by robertgro** [comment]
Indeed. I can confirm this bug in all major browsers (chromium, firefox). Thanks for your report.
